server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/create_mv_vpc-1.0.0.sql

```diff
@@ -21,4 +21,10 @@ CREATE MATERIALIZED VIEW {table_name}_mview AS
           ELSE 'egress'
         END AS `aws.vpc.flow-direction`
 FROM
-    {table_name};
+    {table_name}
+WITH (
+    auto_refresh = 'true',
```

Comment:
any option to configure refresh?

Reply:
Not really, can maybe look at adding it in a future release